jcache/commons/src/main/java/org/infinispan/jcache/SecurityActions.java

```diff
@@ -0,0 +1,54 @@
+package org.infinispan.jcache;
+
+import javax.management.MBeanServer;
+import javax.management.ObjectName;
+import javax.management.QueryExp;
+import java.security.PrivilegedAction;
+import java.util.HashSet;
+import java.util.Set;
+
+import static java.security.AccessController.doPrivileged;
+
+/**
+ * SecurityActions for the org.infinispan.jcache package.
+ * <p>
+ * Do not move. Do not change class and method visibility to avoid being called from other
+ * {@link java.security.CodeSource}s, thus granting privilege escalation to external code.
+ *
+ * @author Ingo Weiss
+ * @since 9.0
+ */
+final class SecurityActions {
```

Comment:
I'd also like to see a test in integrationtests/security-manager-it/ that tests mbean registration